.github/workflows/pr-test-lint.yml

```diff
@@ -9,6 +9,7 @@ on:
       - 'go.sum'
       - 'Makefile'
       - '.github/workflows/pr-test-lint.yml'
+      - 'providers/os/detector/testdata/*.toml'
```

Comment:
We probably want to run on any of the testdata/*.toml changes not just the detector ones

Reply:
lets do that